notary-integration-test/src/main/kotlin/integration/helper/ConfigHelper.kt

```diff
@@ -67,44 +57,37 @@ class ConfigHelper(private val accountHelper: AccountHelper,
         }
     }
 
-    //Creates config for BTC multisig addresses generation
+    /** Creates config for BTC multisig addresses generation */
     fun createBtcPreGenConfig(): BtcPreGenConfig {
+        val btcPkPreGenConfig =
```

Comment:
why do we need this? now we have to load config from file system every time this function is called.